modules/ROOT/pages/workday/workday-connector-config-topics.adoc

```diff
@@ -4,7 +4,7 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../../assets/images/
 
-Configure the Global element for the connector. 
+Configure the Global element for the connector.
```

Comment:
Per our term presentation list, Mule-specific use is Global Element but global, general use is "global element."